app/src/features/surveys/view/SurveyAnimals.tsx

```diff
@@ -535,14 +536,25 @@ const SurveyAnimals: React.FC = () => {
         onNo={() => setOpenRemoveCritterDialog(false)}
         onYes={handleRemoveCritter}
       />
-      <H2ButtonToolbar
-        label="Marked or Known Animals"
-        buttonLabel="Add Animal"
-        buttonTitle="Add Animal"
-        buttonProps={{ variant: 'contained', color: 'primary' }}
-        buttonStartIcon={<Icon path={mdiPlus} size={1} />}
-        buttonOnClick={toggleDialog}
-      />
+      <Toolbar>
+        <Typography
+          component="h2"
+          variant="h4"
+          sx={{
+            flex: '1 1 auto'
+          }}>
+          Marked and Known Animals
+        </Typography>
+        <Button
+          component={RouterLink}
+          to={`animals/${MANAGE_ANIMALS_DEFAULT_URL_PARAM}`}
```

Comment:
I think I understand what this code is for... On the `/animals` page, if it receives a selected critter ID of exactly `0`, then the page renders in some default state; and if instead the page receives some actual critter ID, then the user is able to manage details for that particular critter?

I think passing an out-of-bounds critter ID for this purpose is an anti-pattern. Instead of using a path param, e.g. `/animals/{id}`, I would use a query param instead, which would allow it to be optional. E.g., `/animals` and `/animals?cid=12`

Reply:
I prefer this stratgey, implementing this change now.

Reply:
Fixed